howToCodeWell/code-quiz
# Create unit tests for app fixtures.php

- Fixes #90 

To test load function inside AppFixtures class 
Should we run such command for creating test database and if yes where should we define it :) ?

docker-compose exec api bin/console --env=test doctrine:database:create
